pom.xml

```diff
@@ -20,204 +20,29 @@
-  <developers>
-    <developer>
-      <name>Olivier Barais</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>David Bernard</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Benoit Cornu</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Didier Donsez</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Favio DeMarco</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Christophe Dufour</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Nicolas Harrand</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Sebastian Lamelas Marcote</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Matias Martinez</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Martin Monperrus</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Carlos Noguera</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Renaud Pawlak</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Nicolas Pessemier</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Nicolas Petitprez</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Phillip Schichtel</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Lionel Seinturier</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Stefan Wolf</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Gérard Paligot</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-    <developer>
-      <name>Alcides Fonseca</name>
-      <roles>
-        <role>Contributor</role>
-      </roles>
-    </developer>
-  </developers>
-
-  <scm>
-    <url>https://github.com/INRIA/spoon</url>
-    <connection>scm:git:https://github.com/INRIA/spoon.git</connection>
-    <developerConnection>scm:git:ssh://github.com/INRIA/spoon.git</developerConnection>
-    <tag>HEAD</tag>
-  </scm>
 
   <properties>
     <java.src.version>1.8</java.src.version>
+  </properties>
+  <!--<properties>
```

Comment:
you might want to remove them instead of commenting them

Reply:
Yes. That's some leftover trial and errors.